src/fwd_analyzer.cpp

```
@@ -142,6 +142,15 @@ class InterleavedFwdFixpointIterator final {
         return std::numeric_limits<int>::max();
     }
 
+    int max_call_depth() const {
```

Clang (14.0.6)

[warning] function 'max_call_depth' should be marked [[nodiscard]]

(modernize-use-nodiscard)